lua/lsp-file-operations.lua

```diff
@@ -108,4 +117,19 @@ M.setup = function(opts)
   end
 end
 
+--- The extra client capabilities provided by this plugin. To be merged with
+--- vim.lsp.protocol.make_client_capabilities() and sent to the LSP server.
+M.default_capabilities = function()
+  local config = M.config or default_config
+  local result = {
+    workspace = {
+      fileOperations = {}
+    }
+  }
+  for operation, capability in pairs(capabilities) do
+    result.workspace.fileOperations[capability] = config.operations[operation]
+  end
```

Comment:
We should also add a check here that this capability is enabled in the config. If someone wants to opt out it should be possible.

Reply:
That was the idea. Isn't this what's happening? See line 123:

```lua
local config = M.config or default_config
```
So it's using the user config if available. And then `config.operations[operation]` evaluates to `true`/`false` depending on what the user configured.

Reply:
Oh, yeah, great!